test-harness/modules/mocker/endpoints/mesos.py

```diff
@@ -0,0 +1,219 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) Mesosphere, Inc. See LICENSE file for details.
+
+"""Mesos mock endpoint"""
+
+import logging
+
+from exceptions import EndpointException
+from mocker.endpoints.recording import (
+    RecordingHTTPRequestHandler,
+    RecordingTcpIpEndpoint,
+)
+
+# pylint: disable=C0103
+log = logging.getLogger(__name__)
+
+
+# pylint: disable=R0903
+class MesosHTTPRequestHandler(RecordingHTTPRequestHandler):
+    """A request hander class mimicking DC/OS IAM (Bouncer) operation.
```

Comment:
> But writing a great docstring is a lot of work and requires some love, and if we don't have the time, then it's sometimes better to not add a docstring. 

Totally, but this is not the case here - I would like to make them decent. I had a typo/silly mistake here, but now it's fixed. If you see such mistake anywhere else - please ping me. This is esp. important as the code may eventually be used/extended by community and we need to make the docs really great!